### PR TITLE
DCOS-43820: Mark and translate tab labels

### PR DESCRIPTION
**Pull request by brandonc**
Noticed this translation opportunity while working on adding translation to enterprise plugins.

## Testing

There should be no visual differences or markup changes as a result of merging this PR

### CODE REVIEW

**Comment by nLight** [comment]
Do we need to update messages.json?

**Comment by brandonc** [comment]
@nLight that file conflicts so much that we just want to extract one time after merging all the competing L10n PR's. Everything works without it because the message ids equal the english fallback

File: src/js/mixins/TabsMixin.js [hunk 1]
```diff
@@ -74,9 +75,11 @@ const TabsMixin = {
         onClick={this.tabs_handleTabClick.bind(this, tab)}
         {...attributes}
       >
-        <span className="menu-tabbed-item-label-text">
-          {this.tabs_tabs[tab]}
-        </span>
+        <Trans
+          render="span"
+          className="menu-tabbed-item-label-text"
+          id={this.tabs_tabs[tab]}
```
Comment (nLight) [inline src/js/mixins/TabsMixin.js]: We're translating all labels independently. Do we need this `Trans`

Comment (brandonc) [reply src/js/mixins/TabsMixin.js]: `i18nMark` doesn't translate, it just let's the extractor know about a string that needs translation. You then plug any of those into a Trans component to perform the actual translation.

Comment (nLight) [reply src/js/mixins/TabsMixin.js]: Oh! ok, thanks for explaining this

**Pull request merged**